content/en/agent/logs/advanced_log_collection.md

```diff
@@ -443,6 +443,37 @@ More examples:
 | 2020-10-27 05:10:49.657  | `\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2}\.\d{3}` |
 | {"date": "2018-01-02"    | `\{"date": "\d{4}-\d{2}-\d{2}`                |
 
+### Automatic Multi-line aggregation
+With agent 7.32+ `auto_multi_line_detection` can be enabled which allows the agent to detect common multi-line patterns automatically.
+
+`auto_multi_line_detection` can be enabled globally in the `datadog.yaml`
+
+```yaml
+logs_config:
+  auto_multi_line_detection: true
+```
+
+It can also be enabled or disabled (overriding the global config) per log configuration:
+
+```yaml
+logs:
+  auto_multi_line_detection: true
```

Comment:
This is not a valid logs integration config.

Let's provide an example of a full (simple) logs integration config that's actually valid, and let's provide examples for Docker and Kubernetes as well (with different tabs, like the other sections on this page)